components/core/src/clp/clo/OutputHandler.cpp

```diff
@@ -20,23 +25,26 @@ NetworkOutputHandler::NetworkOutputHandler(std::string const& host, int port) {
 }
 
 ErrorCode NetworkOutputHandler::add_result(
-        std::string const& original_path,
-        std::string const& message,
-        epochtime_t timestamp
+        string_view orig_file_path,
+        string_view orig_file_id,
+        Message const& encoded_message,
+        string_view decompressed_message
 ) {
-    msgpack::type::tuple<std::string, epochtime_t, std::string> src(
-            original_path,
-            timestamp,
-            message
+    msgpack::type::tuple<string, epochtime_t, string, string, size_t> src(
+            orig_file_path,
+            encoded_message.get_ts_in_milli(),
+            decompressed_message,
+            orig_file_id,
+            encoded_message.get_log_event_ix()
```

Comment:
Why add them to the end?

Reply:
The native/search.py in the package doesn't distinguish between if the result is from clp-s or clp. clp has two metadata fields but clp-s only has one. if we don't put them to the end, the decompressed_message will end up having different index (offset?) in the msgpack

Reply:
Then can we reorder clo and clp-s so that it goes:

* timestamp
* decompressed_message
* orig_file_path
* orig_file_id
* log_event_idx

Reply:
pushed

Reply:
You also need to update `clp_s::search::NetworkHandler::write`, right?

Reply:
GG, I did, but in another PR. will add to this PR